packages/core/src/video/VideoForRendering.tsx

```diff
@@ -95,14 +97,15 @@ const VideoForRenderingForwardFunction: React.ForwardRefRenderFunction<
 		frame,
 		absoluteFrame,
 		playbackRate,
+		props.children,
```

Comment:
ESLint warning is probably correct, the dependency array should include a value that makes the useEffect trigger when the children change.

Reply:
Let me check that